subsys/bluetooth/host/Kconfig

```diff
@@ -560,6 +560,13 @@ config BT_DF_CONNECTION_CTE_RX
 	  Enable support for reception and sampling of Constant Tone Extension
 	  in connection mode.
 
+config BT_DF_CONNECTION_CTE_TX
+	bool "Enable support for transmission of CTE in connection mode"
+	depends on BT_DF
+	help
+	  Enable support for transmission of Constant Tone Extension in
+	  connection mode.
+
```

Comment:
Given that we now have 4 configs that `depends on BT_DF`, perhaps we should consider using `if`: https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html?highlight=kconfig#factoring-out-common-dependencies

Reply:
Good point, change applied.